src/helpers/group/group-helper.js

```diff
@@ -35,9 +43,9 @@ export const fetchGroupName = (id) =>
     }
   });
 
-export const fetchFilterGroups = (filterValue) =>
+export const fetchFilterApprovalGroups = (filterValue) =>
   getAxiosInstance().get(`${RBAC_API_BASE}/groups/${filterValue.length > 0
-    ? `?name=${filterValue}`
+    ? `?role_names="Approval Administrator,Approval Approver,"&name=${filterValue}`
```

Comment:
Just a question here. Shouldn't we always use the role names filter? And if sou, I think you can merge the `getRbacApprovalGroups` and `fetchFilterApprovalGroups` functions.